java/runtime/src/main/java/org/ray/runtime/runner/worker/DefaultWorker.java

```diff
@@ -17,6 +17,7 @@
   public static void main(String[] args) {
     try {
       System.setProperty("ray.worker.mode", "WORKER");
+      System.setProperty("ray.run-mode", "CLUSTER");
```

Comment:
Why add this line?

Reply:
@jovany-wang

Reply:
The reason for this change is that the `DefaultWorker` is only used for Cluster mode.

This can correct passing an incorrect `run-mode` item in, otherwise the polymorphic `RayRuntime` would be an instance of `DevRuntime` once we pass in a `singleprocess`.

Reply:
Can we just assert `run mode != single_process` instead?